static/app/views/profiling/landing/profilingTableCell.tsx

```diff
@@ -1,37 +1,80 @@
 import DateTime from 'sentry/components/dateTime';
 import Duration from 'sentry/components/duration';
+import Link from 'sentry/components/links/link';
 import {IconCheckmark, IconClose} from 'sentry/icons';
 import {t} from 'sentry/locale';
+import {Organization, Project} from 'sentry/types';
+import {Trace} from 'sentry/types/profiling/core';
+import {defined} from 'sentry/utils';
 import {Container, NumberContainer} from 'sentry/utils/discover/styles';
+import {getShortEventId} from 'sentry/utils/events';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjects from 'sentry/utils/useProjects';
 
 import {TableColumn, TableDataRow} from './types';
 
-function ProfilingTableCell(
+function renderProfilingTableCell(
```

Comment:
@Zylphrex one last small change -> I would move this to the table component and only export {ProfilingTableCell} from here. This way the "API" is clear, we are exporting a react component and it's up to the table to handle how to render it.